packages/nextjs-experimental/src/app/basic-chat/page.tsx

```diff
@@ -0,0 +1,92 @@
+/** @jsxImportSource ai-jsx/react */
+import { Chat, Conversation, Message, ChatState, sendMessage, SerializedMessage } from 'ai-jsx/experimental/next/chat';
+import { ChatCompletion, SystemMessage } from 'ai-jsx/core/completion';
+import ResultContainer from '@/components/ResultContainer';
+import { ReactNode, Suspense } from 'react';
+
+async function onSend(formData: FormData, conversation?: SerializedMessage[]) {
+  'use server';
+
+  return sendMessage(
```

Comment:
```suggestion
  return sendChatMessage(
```

I think this name is a little more specific/clear; IMO `sendMessage` feels too close to the `postMessage` API, which is a totally different thing.